README.md

```diff
@@ -77,7 +77,17 @@ More precisely:
 
 2. The Core Team will regularly review `swift-evolution` pull requests, and provide feedback to the authors in the pull request on whether or not the proposal looks within reason of something that might be accepted.
 
-3. If a proposal gets a positive indicator from the Core Team for later review, the authors must provide an implementation prior to the proposal being formally reviewed.  An implementation should be provided in the form of a pull request against the impacted repositories (e.g., `swift`, `swift-package-manager`), and the proposal should be updated with a link to that pull request.  The existence of an implementation does not guarantee that the proposal will be accepted, but it is instrumental in evaluating the quality and impact of the proposal.
+3. If a proposal gets a positive indicator from the Core Team for later review, the authors must provide an implementation prior to the proposal being formally reviewed.  An implementation should be provided in the form of a pull request against the impacted repositories (e.g., `swift`, `swift-package-manager`), and the proposal should be updated with a link to that pull request.  It is expected that before review begins the pull request implementation runs at least the following tests:
```

Comment:
The language here says "runs at least" the following tests, but doesn't set expectations.  Should all the tests complete successfully?  Certainly for the Source Compatibility suite a proposal may deliberately create source-breaking changes, so it is expected to fail.  But in those circumstances, the information from running the tests are important.  I suggest we put more clarity into these instructions on what the outcome of running these tests should imply when advocating a proposal run for a review.